README.md

```diff
@@ -290,6 +290,25 @@ The most commonly adjusted settings are probably
 
 Also, you can add additional vocabulary words, words to replace, search engines and more. Complete the community setup instructions above, then open the `settings` folder to see the provided CSV files and customize them as needed.
 
+## Overriding settings without forking
+
+To override a setting without forking the community repository, create talon files with more specific tags than the default. This customizes your setup without affecting the base repository. Any tag will work. For example, setting the language on a talon file or list will override the community settings. This is useful for personal customizations or workflow-specific tweaks.
```

Comment:
```suggestion
You can override Talon lists by creating a new `.talon` file of your own, rather than changing the existing list in the repository.

This reduces how much manual `git merge`-ing you'll have to do in the future, when you go to merge new versions of this repository (colloquially called "upstream") with your local changes. This is because _new_ files you create will almost never conflict with upstream changes, whereas changing an existing files (especially hot spots, like commonly-customized lists) frequently do.

Your override files can even live outside of the `community` repository (anywhere in `~/.talon/user`), if you prefer, further simplifying merging.

To do so, simply create a `.talon` file with a more specific [context header](https://talon.wiki/Customization/talon-files#context-header) than the default. (For example, `lang: en` or `os: mac` main). Talon ensures that the most precise header (your override file) wins.
```

Reply:
I think what you're referring to as "tags" is best described as the context header, which can include tags, but also other matchers, like `os`, that are not tags.

Reply:
Agree tags is just a subset of the available matchers.

Also in the suggestion it should probably be a new `.talon-list` file

Also `~/.talon` doesn't exist on windows machines. Something like `TALON_HOME` is probably a better way of referring to it.

Reply:
> Also ~/.talon doesn't exist on windows machines. Something like TALON_HOME is probably a better way of referring to it.

Good point. We can just say "Talon home directory", per https://talonvoice.com/docs/#getting-started.

Reply:
I updated to match this text.